README.md

```
@@ -1,39 +1,100 @@
+- **InMemoryDatastore** - Full in-memory implementation for testing
+- **Upload Progress Tracking** - Real-time progress streams with pause/resume/cancel
+- **Slow Mode Simulation** - Configurable upload chunking for realistic testing
+- **HTTP Client** - Built-in HTTP server to serve stored data
+- **Multiple Instances** - Named datastore instances for isolation
 
-## Features
+## Usage
 
-TODO: List what your package can do. Maybe include images, gifs, or videos.
+This is the base interface package, included are two reference implementations.
+** Concrete implemenations for different providers are provided in other packages. **
```

markdownlint-cli2 (0.17.2)

Spaces inside emphasis markers
null

(MD037, no-space-in-emphasis)